### ISSUE TITLE
My LaTeX strings look bold when exporting in .pdf and .eps

### ISSUE THREAD

**Issue by Ipuch** [plotly/Kaleido#196]
I'm using plotly to generate my figures for a research article, in pycharm. I need to export my figure in **.eps or .pdf** and I have an issue that doesn't appear if I export in png.

LaTeX strings got bold when exporting the figure. I don't know how to resolve the problem. Do you know any command to remove this bold effect?

![image](https://user-images.githubusercontent.com/40755537/181358205-4b351849-ae70-4f47-9ae2-b3659a240759.png)
[q_integrated.pdf](https://github.com/plotly/plotly.py/files/9203080/q_integrated.pdf)

Here is the python code to write the labels for example:

```python fig.update_yaxes(row=1, col=1, title=r"$\text{q (m)}$")
for i in range(2, rows + 1):
    fig.update_yaxes(row=i, col=1, title=r"$\text{q (degree)}$")
for i in range(1, cols + 1):
    fig.update_xaxes(row=rows, col=i, title=r"$\text{Time (s)}$")

#This is how I export the figure:

python fig.write_image(out_path_file + "/q_integrated.png")
fig.write_image(out_path_file + "/q_integrated.pdf")
fig.write_image(out_path_file + "/q_integrated.eps")

**Comment by gvwilson** [comment]
Thanks for your interest in Kaleido. We are currently working on an overhaul that might address your issue - we hope to have news in a few weeks and will post an update then. Thanks - @gvwilson

**Comment by YDS67** [comment]
Any news on this? Still having this issue in Plotly 5.24.1 and Kaleido 0.2.1 on Ubuntu 24.04.1

Is the fork created by @konmenel still the only relevant solution?

I also want to emphasize how much I'd like to use Plotly for my scientific papers and conferences. And just in general, keeping my plots as pdf figures is vastly superior to keeping them as png.

**Comment by Ipuch** [comment]
Could pushing your fork on conda or somewhere else be a solution ?

**Comment by konmenel** [comment]
I have never published anything with conda before so I don't know how to do that. I have a feeling that this will take a long time because the code would need to be compiled for all supported platforms and architectures. I might have a look, however, I think the best solution would be to implement the change with the new version of Kaleido.

**Comment by gvwilson** [comment]
Thanks for the discussion - we plan to switch plotly.py to the new version of Kaleido (that uses an external Chrome binary) in a point release some time in February, and I hope to have more news (including a firmer date) by the end of this week. Thanks - @gvwilson

**Comment by dajerlont** [comment]
Alternatively, and while we wait on news from @gvwilson, and the actual update to come out, you can:

1. Export the Plotly figure to PDF.
2. Open with Inkscape.
3. Inside [Inkscape](https://inkscape.org/), select all the LaTeX objects and set their stroke to none.
4. Save the PDF from Inkscape.

This is not optimal, and may produce other complications, but it's faster than @konmenel's solution (however, **LONG** that long is).

**Comment by konmenel** [comment]
As suspected the issue is even easier to fix than before. It is literally an extra line. I created a new PR #243 in the new version.

In the meantime, if anyone is interested the new kaleido is very easy to install. Just clone the branch "mathjax_config_new" of my fork and install with `pip` from the root of Kaleido
```bash
pip install src/py
```
However, I should note that this version is in pre-release. This solution is temporary. I already encountered two issues which I list in the PR. Hopefully, all will be fixed when the new version is released.

**Comment by DouwMarx** [comment]
@dajerlont I have not been able to unbold the text using your suggested temporary fix. I followed these steps:
- Opening exported pdf in Inkscape 1.4 (e7c3feb100, 2024-10-09)
- Select Latex-based text by clicking on it
- Click on Fill and Stroke -> Fill -> x (no paint x button)
- Save / Export as pdf

Is there something I am missing? Thanks for your help.

**Comment by ayjayt** [comment]
I'll try to give this a permanent solution tomorrow!

**Comment by dajerlont** [comment]
@ayjayt I am very excited for this!

@DouwMarx, let me walk you through how I do it with Inkscape.

1. Open the PDF with Inkscape.
2. Select the LaTeX objects. You should see that the objects have both a fill and stroke color (first screenshot). 

![Image](https://github.com/user-attachments/assets/0e50e83a-4f5f-4dfd-bf61-4d6a4ca1baa1)

3. While holding the shift key, select the X for no color for the stroke (second screenshot). The stroke should now be set to None.
4. Now you can export/save as PDF.

![Image](https://github.com/user-attachments/assets/a75e0a59-60ee-4b25-befb-0a7794c20323)

I have not always had luck with Inkscape. Sometimes the LaTeX objects will disappear once the PDFs are opened or saved with Inkscape. I'm not sure what kind of problem you're having.

**Comment by ayjayt** [comment]
Alright, I just wanna do some tests and then merge this https://github.com/plotly/Kaleido/pull/274, thanks again @konmenel!
